opentelemetry-exporter-gcp-trace/setup.cfg

```diff
@@ -26,8 +26,8 @@ package_dir=
 packages=find_namespace:
 install_requires =
     google-cloud-trace ~= 1.1
-    opentelemetry-api ~= 1.0
-    opentelemetry-sdk ~= 1.0
+    opentelemetry-api ~= 1.4
+    opentelemetry-sdk ~= 1.4
```

Comment:
My initial thinking was to maintain a universal minimum since these packages are updated/released together. In retrospect, I think this just places unnecessary constraints on version compatibility. Will update.